src/Client/V2/Access/NavigationAccess.cs

```diff
@@ -0,0 +1,173 @@
+using System.Threading.Tasks;
+using EmsApi.Dto.V2;
+
+namespace EmsApi.Client.V2.Access
+{
+    /// <summary>
+    /// Access to airport navigation information: airports, runways, procedures, segments, waypoints, and NAVAIDs. 
+    /// </summary>
+    public class NavigationAccess : RouteAccess
+    {
+        /// <summary>
+        /// Gets the active airports for the EMS system.
+        /// </summary>
+        /// <param name="context">
+        /// The optional call context to include.
+        /// </param>
+        public virtual Task<NavigationAirport[]> GetAirportsAsync( CallContext context = null )
```

Comment:
Most everything else returns `IEnumerable`s instead of arrays so I would continue that here if possible

Reply:
Good call. I switched these all to IEnumerables.